src/components/Plugin/CommentBox/ReplyBlock.vue

```diff
@@ -0,0 +1,254 @@
+<script setup>
+import { defineProps, watch, computed, ref, defineEmits } from 'vue';
+import { useNotifications } from '@/composables/useNotifications';
+import { useModal } from '@/composables/useModal';
+import { useWeb3 } from '@/composables/useWeb3';
+import { signMessage } from '@snapshot-labs/snapshot.js/src/utils/web3';
+import { getInstance } from '@snapshot-labs/lock/plugins/vue3';
+const auth = getInstance();
+const { modalOpen, modalAccountOpen } = useModal();
+const { web3 } = useWeb3();
+const web3Account = computed(() => web3.value.account);
+const props = defineProps({
+  item: Object,
+  space: Object,
+  profiles: Object,
+  mainThread: String,
+  proposal: Object
+});
+const emit = defineEmits([
+  'deleteItem',
+  'updateItem',
+  'replyComment',
+  'scrollTo'
+]);
+const toggleComment = ref(true);
+const toggleReplyTo = ref(false);
+const toggleEditComment = ref(true);
+const loading = ref(false);
+const threeDotItems = computed(() => {
+  const items = [
+    { text: 'edit', action: 'edit' },
+    { text: 'delete', action: 'delete' }
+  ];
+
+  return items;
+});
+const closeModal = ref(false);
+function selectFromThreedotDropdown(e) {
+  if (e === 'edit') {
+    toggleEditComment.value = false;
+    toggleComment.value = true;
+  }
+  if (e === 'delete') {
+    if (!web3Account.value) return (modalAccountOpen.value = true);
+    closeModal.value = true;
+  }
+}
+async function deleteData(url = '', data = {}, authorization) {
+  // Default options are marked with *
+  const response = await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-type': 'application/json;charset=UTF-8',
+      ...authorization
+    }
+  });
+  return response.json(); // parses JSON response into native JavaScript objects
+}
+const { notify } = useNotifications();
+async function deleteItem() {
+  if (loading.value) return;
+  try {
+    loading.value = true;
+    const token = sessionStorage.getItem('token');
```

Comment:
Can you use localStorage instead of session here? Also add a prefix like `_commentBox.token`

Reply:
ok resolved, **token** now become **_commentBox.token** and **sessionStorage** become **localStorage** and i also fix some of the UI issue